README.md

```diff
@@ -25,13 +25,18 @@ Here are some examples of how you can use the PV211 Utils library:
   [![Open in Colab][colab-badge]][cranfield]
```

Comment:
The MAP score is suspect and should be measured again after all the planned changes (using `TfIdfSystem`, `preprocessing.preprocessing`, etc.).